src/modules/sensors/vehicle_air_data/VehicleAirData.cpp

```diff
@@ -189,6 +184,8 @@ void VehicleAirData::Run()
 						}
 
 						ParametersUpdate(true);
+						_calibration[uorb_index].set_device_id(report.device_id);
+						_calibration[uorb_index].ParametersSave(uorb_index);
```

Comment:
you might want to call a `param_notify_changes()` outside the loop if a device id got updated

Reply:
Also, this should probably be done before `ParametersUpdate(true)` to avoid triggering a double param update

Reply:
Shouldn't we only save if the device_id really changed? Otherwise we'll have a param save at every boot